tailwind.config.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/extensions */
+/* eslint-disable global-require */
```

Comment:
maybe exclude this file and others config files from eslint ?

Reply:
It just doesn't work
<img width="913" alt="Screenshot 2022-06-06 at 22 47 51" src="https://user-images.githubusercontent.com/32927775/172246312-1dc1b93b-5af3-488e-acdc-3557358c2cda.png">
<img width="1840" alt="Screenshot 2022-06-06 at 22 48 01" src="https://user-images.githubusercontent.com/32927775/172246350-3c6a03bd-39a1-4c55-9131-d73f6e3a0131.png">